src/transaction/log_2pc.c

```diff
@@ -1422,18 +1425,16 @@ log_2pc_read_prepare (THREAD_ENTRY * thread_p, int acquire_locks, log_tdes * tde
     }
 }
 
+// *INDENT-OFF*
 void
 log_2pc_read_prepare (THREAD_ENTRY * thread_p, int acquire_locks, log_tdes * tdes, log_reader & log_pgptr_reader)
 {
-  LOG_REC_2PC_PREPCOMMIT *prepared;	/* A 2PC prepare to commit log record */
+  const LOG_REC_2PC_PREPCOMMIT *prepared;	/* A 2PC prepare to commit log record */
   LK_ACQUIRED_LOCKS acq_locks;	/* List of acquired locks before the system crash */
   int size;
 
   log_pgptr_reader.advance_when_does_not_fit (sizeof (*prepared));
-
-  // *INDENT-OFF*
-  prepared = const_cast<LOG_REC_2PC_PREPCOMMIT*> (log_pgptr_reader.reinterpret_cptr<LOG_REC_2PC_PREPCOMMIT> ());
-  // *INDENT-ON*
+  prepared = log_pgptr_reader.reinterpret_cptr<LOG_REC_2PC_PREPCOMMIT> ();
```

Comment:
could even be `const ... *const prepared ...` but given it's a pure C file, maybe stick to C conventions

Reply:
Could be, but I'd rather keep the current code. Imo, making a variable that doesn't change const is optional, unless it is meant not to change under any circumstances :). The value of the pointer here is not meant to be changed under any circumstances, the pointer itself could be changed.